app/[locale]/error.tsx

```diff
@@ -6,11 +6,13 @@ import { useTranslations } from 'next-intl';
 import type { FC } from 'react';
 
 import Button from '@/components/Common/Button';
+import { useHideSidebar } from '@/hooks';
 import CenteredLayout from '@/layouts/Centered';
 
 const ErrorPage: FC<{ error: Error }> = ({ error }) => {
   captureException(error);
   const t = useTranslations();
+  useHideSidebar();
```

Comment:
why need that on error and 404 ?

Reply:
Since the sidebar is moved into base layout, we must hide it in error and 404 pages. I tried multiple ways but settled on using state and context.